### PR TITLE
add width and height to log

### PR DESCRIPTION
**Pull request by Ben-Epstein**
width and height are necessary for OD dataframes

relates to https://app.shortcut.com/galileo/story/5206/edits-cart-fix-coordinates-format

### CODE REVIEW

File: dataquality/loggers/data_logger/object_detection.py [hunk 1]
```diff
@@ -83,13 +89,15 @@ def _get_input_df(self) -> DataFrame:
...
+            width=self.width,
+            height=self.height,
             **self.meta,
             # bbox=self.bbox,
             # cls=self.cls,
         )
         if self.split == Split.inference:
             inp["inference_name"] = [self.inference_name] * df_len
-
+        # import pdb; pdb.set_trace()
```
Comment (anthonycorletti) [inline dataquality/loggers/data_logger/object_detection.py:100]: 🧹

**Review by anthonycorletti — APPROVED** [review]

File: dataquality/loggers/data_logger/object_detection.py (hunk 1 above)
Comment (bogdan-galileo) [inline dataquality/loggers/data_logger/object_detection.py:100]: remove this ?

**Review by anthonycorletti — APPROVED** [review]

File: dataquality/loggers/data_logger/object_detection.py [hunk 2]
```diff
@@ -61,10 +63,14 @@ def __init__(
         split: Optional[str] = None,
         meta: Optional[MetasType] = None,
         inference_name: Optional[str] = None,
+        width: Optional[List[float]] = None,
+        height: Optional[List[float]] = None,
```
Comment (bogdan-galileo) [inline dataquality/loggers/data_logger/object_detection.py:67]: You use `int` as their type below, should these match ?

File: dataquality/integrations/ultralytics.py [hunk 3]
```diff
@@ -370,6 +370,8 @@ def convert_dataset(self, dataset: Any) -> List:
                     ODCols.image: str(file_path),
                     ODCols.bbox: bbox_gold,
                     ODCols.gold_cls: image["cls"],
+                    ODCols.width: width,
```
Comment (bogdan-galileo) [inline dataquality/integrations/ultralytics.py:373]: I think we should set `width` and `height` from `shape = image["ori_shape"]` see a few lines above https://github.com/rungalileo/dataquality/blob/b04261ce265f2c4f613ced91e9d17f64c30fbe16/dataquality/integrations/ultralytics.py#LL350C13-L350C39.

The ones you're setting here are probably the w/h of the batch which I think is the max of any image in the batch

File: dataquality/loggers/data_logger/object_detection.py (hunk 2 above)
Comment (Ben-Epstein) [reply dataquality/loggers/data_logger/object_detection.py:67]: @bogdan-galileo that is the type for the column name/index. This is the value

File: dataquality/integrations/ultralytics.py (hunk 3 above)
Comment (Ben-Epstein) [reply dataquality/integrations/ultralytics.py:373]: great catch!!

Comment (Ben-Epstein) [reply dataquality/integrations/ultralytics.py]: updated

**Review by bogdan-galileo — APPROVED** [review]
LGTM, thanks!!

**Pull request merged**